application/ApplicationRequest.js

```diff
@@ -26,10 +26,12 @@ class Request {
                 return response.data;
             } else if (request == 'GetAllNodes') {
                 return response.data.data;
+            } else if(request == 'GetServerInfo') {
+                return response.data.data
             }
         }).catch(error => {
-            let err = createError(request, error, data);
```

Comment:
Why are you removing the data variable, and removing ; on the end of the throw error lines?

Reply:
i'm not removing in the NPM version don't have this...

Reply:
The NPM is pulled directly from this code, so anything included here will be included in NPM

Reply:
Same thing with over convo, use github version :)